src/vaadin-menu-bar-buttons-mixin.html

```diff
@@ -169,6 +209,56 @@
       }
     }
 
+    render() {
+      this.__renderButtons(this.items);
```

Comment:
If I call the public `render` function when the ellipsis button is visible, the buttons will overflow.

The overflowing should be taken into account if we want this function to be public.

Reply:
Fixed.